webcommon/libs.graaljs/external/js-23.0.3-license.txt

```diff
@@ -2,8 +2,8 @@ Name: Graal SDK and Truffle API
 Description: Graal SDK and Truffle API
```

Comment:
Nit: please rename the file to `...-24.0.0-...`.

Reply:
Fixed in #7253